gapis/api/vulkan/state_rebuilder.go

```diff
@@ -1498,20 +1501,24 @@ func (sb *stateBuilder) createImage(img ImageObjectʳ, imgPrimer *imagePrimer) {
 	} else {
 		// Otherwise, we have no sparse bindings, we are either non-sparse, or empty.
 		if !isDenseBound(img) {
-			return
+			return nil, fmt.Errorf("no sparse bindings found")
```

Comment:
This is not an error IIRC. This means we are sparse bound, but have no bindings, this should be totally valid.